src/main/target/common_post.h

```diff
@@ -25,3 +25,11 @@
 # undef VTX_SMARTAUDIO
 # undef VTX_TRAMP
 #endif
+
+#if defined(SPEKTRUM_BIND_PIN) && !defined(SPEKTRUM_BIND)
+#define SPEKTRUM_BIND
+#endif
+
+#if !defined(SPEKTRUM_BIND_PIN) && defined(SPEKTRUM_BIND)
```

Comment:
I agree with @martinbudden here. Using `SPECTRUM_BIND_PIN` as the conditional is not 'pretty', but using two different defines introduces room for inconsistencies, and requires scaffolding code to stop it from coming apart. (i.e. it's not intuitive what the build system should do if `SPECTRUM_BIND_PIN` is defined, but not `SPECTRUM_BIND`, and choosing an arbitrary way to handle it isn't a good solution).